src/utils/compute-class-names.ts

```diff
@@ -0,0 +1,10 @@
+export default function computeClassNames(deps: Record<string, unknown>) {
```

Comment:
Propongo eliminar el (` ${key}`), en su lugar utilicemos solo el `key` y en el `joing(' ')` agregar un espacio en blanco, de esta forma evitamos dejar un espacio en blanco al principio y asi evitamos tener la clase pegada al uso de esta funcion mejorando un poco la legibilidad del codigo.

Ejemplo:
 - Con espacio al principio: `nombre-clase${classNames}`
 - Sin espacio al principio: `nombre-clase ${classNames}`

```suggestion
export default function computeClassNames(deps: Record<string, unknown>) {
  const classes = Object.entries(deps)
    .map((item) => {
      const [key, value] = item;
      return key && !!value ? key : '';
    })
    .join(' ').trim();

  return classes.length ? classes : '';
}
```

Reply:
De acuerdo, mejora la legibilidad.